cookbooks/clearwater/recipes/vellum.rb

```diff
@@ -32,11 +32,7 @@
 # under which the OpenSSL Project distributes the OpenSSL toolkit software,
 # as those licenses appear in the file LICENSE-OPENSSL.
 
-package "vellum-node" do
-  action [:install]
-  options "--force-yes"
-end
-package "homer-cassandra" do
```

Comment:
Should we be removing memento-cassandra as well? Or should that only get installed if memento is included?